src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGiftcardComponent.js

```diff
@@ -44,18 +21,54 @@ function getGiftCardElements() {
   };
 }
 
+function renderGiftCardSelectForm() {
+  const { giftCardSelect, giftCardUl } = getGiftCardElements();
+  if (giftCardUl?.innerHTML) {
+    return;
+  }
+
+  const { paymentMethodsResponse } = store.checkout;
+  const { imagePath } = store.checkoutConfiguration.session;
+
+  const giftCardBrands = paymentMethodsResponse.paymentMethods.filter(
+    (pm) => pm.type === constants.GIFTCARD,
+  );
+
+  giftCardBrands.forEach((giftCard) => {
+    const newListItem = document.createElement('li');
+    newListItem.setAttribute('data-brand', giftCard.brand);
+    newListItem.setAttribute('data-name', giftCard.name);
+    newListItem.setAttribute('data-type', giftCard.type);
+
+    const span = document.createElement('span');
+    span.textContent = giftCard.name;
+    const img = document.createElement('img');
+    img.src = `${imagePath}${giftCard.brand}.png`;
+    img.width = 40;
```

Comment:
Will this cause issues on mobile?

Reply:
This was a request from Cami, thats how the checkout team is doing for PMs as well.